sdk/confidentialledger/azure-confidentialledger/azure/confidentialledger/_operations/_operations.py

```diff
@@ -19,8 +19,7 @@
 )
 from azure.core.paging import ItemPaged
 from azure.core.pipeline import PipelineResponse
-from azure.core.pipeline.transport import HttpResponse
-from azure.core.rest import HttpRequest
+from azure.core.rest import HttpRequest, HttpResponse
```

Comment:
Since this is all generated code instead of manually changing all of these - you can regenerate from the swagger with autorest and it will update with the correct formatting of azure.core.rest

Reply:
unless there was a reason for not regenerating ^ :)

Reply:
I tried regenerating but it produced so many changes it was looking like there was a risk of introducing unintended consequences. As this is only to help with the linting in builds I did the least amount of changes to mitigate it.